core/src/lib.rs

```diff
@@ -293,38 +298,37 @@ impl Schema {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
 
-    pub fn as_edn_value(&self) -> edn::Value {
+    pub fn to_edn_value(&self) -> edn::Value {
         let mut all_values = Vec::new();
         for (entid, attribute) in &self.schema_map {
             let mut attribute_map: BTreeMap<edn::Value, edn::Value> = BTreeMap::default();
-            attribute_map.insert(edn::Value::NamespacedKeyword(NamespacedKeyword::new("db", "id")), edn::Value::Keyword(Keyword::new(entid.to_string())));
-
             let some_ident = self.get_ident(entid.clone());
```

Comment:
You probably don't need to clone the `entid` here; `get_ident` should take it by reference.

It is more idiomatic to pattern match here, like:
```rust
if let Some(ref ident) = self.get_ident(entid) {
    ...
} else {
    ...
}
```
That way you don't need to fetch-test-unwrap.  Try it and see!